### ISSUE TITLE
Funding with Tidelift

### ISSUE THREAD

**Issue by jodastephen** [mockito/mockito#1742]
I noticed you currently run adverts on your home page and ask for feedback for alternatives. One option is [Tidelift](https://tidelift.com), where Mockito could earn [$417 per month](https://tidelift.com/lifter/search/maven/org.mockito:mockito-core) at present. Active maintainers have to sign up individually, but income can be [split](https://tidelift.com/docs/lifting/paying). For info, I'm using Tidelft for the [Joda](https://tidelift.com/subscription/pkg/maven-joda-time-joda-time?utm_source=maven-joda-time-joda-time&utm_medium=referral&utm_campaign=website) projects and its been effective so far.

**Comment by TimvdLippe** [comment]
@raphw I recall you having some thoughts on this?

**Comment by bric3** [comment]
Hi @jodastephen, have you tried Github sponsoring feature?

**Comment by mockitoguy** [comment]
I was meaning to try it out. I think we should. We can split up for active maintainers. Perhaps this would motivate us for putting more effort.

**Comment by TimvdLippe** [comment]
I am not sure if tax-wise and employer-wise I can accept money from an open collective. I would need to figure that out.

**Comment by jodastephen** [comment]
@bric3 I have the GitHub sponsor button enabled (links to Tidelift), but I'm not a member of their sponsorship scheme.

**Comment by raphw** [comment]
I am using Tidelift for Byte Buddy and it works perfectly. I had suggested it in the developer Slack channel before but I do not feel entitled to start it but would rather want @mockitoguy to take the lead on this.

If we split it, I am happy to take a cut for the time I am contributing actively. It is not the world but it feels like a token of appreciation. At a minimum, it could pay for the running expenses such as the webpage.

**Comment by TimvdLippe** [comment]
While I do appreciate the "contributions for time actively maintained", how do we quantify that? I think a flat split is easier. And if you can't commit the time, maybe just become an emeritus maintainer.

**Comment by raphw** [comment]
I agree that we need to have a good definition of *active contributor* but I do neither know how we should decide this fairly. I am not very active these days since I do not see many things in my domain to improve right now but I have spent a few days of my time to add support for modules not that long ago. I hope we do not need to introduce a time tracker to make this work ;)

That said, I think @mockitoguy should get a fair share no matter what as he was leading the project's inception, that is just fair.

**Comment by bric3** [comment]
I agree entirely with @raphw. On m’y part I’m focusing strongly on family matters during my spare time at this time so I don’t contribute more than few comment since quite a while to be counted as an _active_ maintainer.

**Comment by TimvdLippe** [comment]
Rather than trying to determine "a fair share", I would rather keep commitments voluntary, as they have been up to this point. Instead, I would vouch for letting it run our server cost and that's about it. I fear that if we put money in this project, it might have an adversarial effect.

**Comment by bric3** [comment]
Unfortunately there is already ads. But you raise an interesting point that link to the essence of open source software. Money is always a hard thing, Maybe this money can be donated to some animal care or child care associations instead.

**Comment by TimvdLippe** [comment]
+1 to donating to a charity.

**Comment by mockitoguy** [comment]
> +1 to donating to a charity.

Great idea!

**Comment by raphw** [comment]
That's a good point, much better then letting it go to waste. One issue is probably that it's taxable income, unless maybe it is paid directly to a foundations account.

**Comment by TimvdLippe** [comment]
I do have some concerns around language like 

> To get professional-grade assurances and long-term maintenance on their open source stack, companies and software teams can buy the Tidelift Subscription to cover all of the packages they use.

There is a lot of information at https://tidelift.com/docs/lifting/tasks-overview with legal stuff at https://tidelift.com/docs/lifting/agreement

Before we sign anything, we should have a lawyer take a look at this. However, I am now more wary of Tidelift. Not so much the terms, but more that if we sign something now, we contractually obligate future maintainers to adhere to these rules; and they might not (want to).

Therefore, I am less enthusiastic, simply because of the (legal) consequences and the impact it can have on this project. If a lawyer can take a look and provide us with more comfort, I am of course open to an agreement. However, lawyers are expensive and I don't think we want to pay money for that now 😢

**Comment by jodastephen** [comment]
Since it is an individual that contracts with Tidelift not the project, and you can terminate at any time, I don't really see how you are obligating future maintainers at all. I'd strongly suggest one or more of you getting on a video call with them to ask questions and discuss your concerns - they are friendly people and have lots of open source project experience.

**Comment by mockitoguy** [comment]
I've read the docs and a couple of articles and I don't think signing up puts other maintainers or a project at risk.

**Comment by raphw** [comment]
You can cancle it any month, it is a very light obligation, e.g. to update release notes on their site which is something we should integrate into the CI. Or to report security issues which is rarely relevant to us.

**Comment by mockitoguy** [comment]
Update: I'm making progress on this, will get back soon.
